lib/guardian_onetime.ex

```diff
@@ -1,4 +1,4 @@
-defmodule Guardian.Token.OneTime do
+defmodule GuardianOneTime do
```

Comment:
I'm not for this module change.  👎  

The previous module was meant to match with how namespacing works for the existing `Guardian.Token.Jwt`

Reply:
I prefer if you keep everything under Guardian namespace as well